docs/content/docs/connectors/table/hive/hive_read_write.md

```diff
@@ -208,7 +208,9 @@ Users can do some performance tuning by tuning the split's size with the follow
 
 ### Read Table Statistics
 
-To obtain hive table statistics faster, When hive table format is `orc` or `parquet`. You can use `table.exec.hive.read-format-statistics.thread-num` to configure the thread number. The default value is the number of available processors in the current system and the configured value should be bigger than 0.
+When the table statistic is not available from Hive metastore, such as `orc` or `parquet`. We will then try to get the statistic by scanning the table. 
```

Comment:
Suggestion:
When the table statistic is not available from the Hive meta store, Flink will try to scan the table to get the statistic to generate a better execution plan. It may cost some time to get the statistic.